docs/src/pages/demos/text-fields/text-fields.md

```diff
@@ -108,6 +108,8 @@ or
 
 You can use third-party libraries to format an input.
 You have to provide a custom implementation of the `<input>` element with the `inputComponent` property.
+The provided input component should handle the `inputRef` property.
+The property should be called with a value implementing the `HTMLInputElement` interface.
```

Comment:
```suggestion
The property should be called with a value implementing the [`HTMLInputElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement) interface.
```
or https://html.spec.whatwg.org/multipage/input.html but the MDN docs are IMO more useable.